scoring.js

```diff
@@ -0,0 +1,79 @@
+const calculateScore = (player) => {
+    switch (player.position) {
+        case 'QB':
+            return calculateTotalPassing(player) + calculateTotalRushing(player)       
+
+        case 'RB': 
+            return   calculateTotalRushing(player) + calculateTotalReceiving(player) + calculateTotalKickReturn(player) + calculateTotalPuntReturn(player)
+
+        case 'WR': 
+            return   calculateTotalRushing(player) + calculateTotalReceiving(player) + calculateTotalKickReturn(player) + calculateTotalPuntReturn(player)
```

Comment:
One cool thing, since these are the same calculation, is to use `switch`'s run-to-end behavior to our advantage

```JavaScript
switch (player.position) {
  case 'QB':
    return calculateTotalPassing(player) + calculateTotalRushing(player)       
  case 'RB': 
  case 'WR': 
    return  calculateTotalRushing(player) + calculateTotalReceiving(player) + calculateTotalKickReturn(player) + calculateTotalPuntReturn(player)
  case 'TE': 
    return    calculateTotalReceiving(player)
  default:
    return 0 
}
```

When it matches RB, since there is no return or break, it will run right into WR which is the calculation you want so it works.